easybuild/easyblocks/p/psi.py

```diff
@@ -147,6 +147,15 @@ def configure_step(self):
             if get_software_root('impi'):
                 self.cfg.update('configopts', " -DENABLE_CSR=ON -DBLAS_TYPE=MKL")
 
+            if self.cfg.name == "PSI4":
```

Comment:
yes, but I'm not gonna touch the old PSI anymore. And it's version hell anyway if I do that.